m4_sessions/1705-inning/group_four.md

```diff
@@ -1,10 +1,31 @@
-# Group Four
+# Lending Privilege
+## Plan
 
-## List your group members here:
-- Katy Farmer
-- Edilene Da Cruz
-- Staci McWilliams
+* 20 - Videos
+* 15 - Guest Speaker(pending) || Individual Reading and Reflection
+* 20 - Small Group Discussion
+* 5 - Share Out 
+
+## Videos
+* [Intro video](https://www.youtube.com/watch?v=SqL9onVybW0) 2:53
+* [The Power of Privledge](https://www.youtube.com/watch?v=N0acvkHIiZs) 14:57
+
+## Guest Speaker || Reading
+* Pending Guest Speaker (estimated time 20 minutes) 20:00
```

Comment:
Did you end up confirming someone?  It can be helpful to have them write out a few bullet points that they will hit during the talk.  Then you can connect the key points together.